pkg/proxy/ipvs/proxier.go

```diff
@@ -1481,6 +1494,7 @@ func (proxier *Proxier) syncService(svcName string, vs *utilipvs.VirtualServer,
 	// bind service address to dummy interface even if service not changed,
 	// in case that service IP was removed by other processes
 	if bindAddr {
+		glog.V(4).Infof("bind addr %s", vs.Address.String())
```

Comment:
Nit: log message should be started with a upper-case word. So, Bind addr...

Reply:
thanks for review, will modify this